usr/share/rear/conf/default.conf

```diff
@@ -2767,6 +2767,10 @@ BLOCKCLONE_SOURCE_DEV=""
 BLOCKCLONE_PROG=""
 # Options that should be passed to BLOCKCLONE_PROG
 BLOCKCLONE_PROG_OPTS=""
+# Do we want to compress the image with gzip (.gz suffix), xz (.xz suffix), bzip2 (.bz2 suffix), zstd (.zst suffix)
+BLOCKCLONE_PROG_COMPRESS=""
```

Comment:
Maybe better named `BLOCKCLONE_COMPRESS_WITH` or `BLOCKCLONE_COMPRESSION_PROGRAM` to indicate that this is actually a compression program and not a standard?